<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mode_Select_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>481</width>
     <height>291</height>
    </rect>
   </property>
   <widget class="QPushButton" name="guard_Mode_Button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>경계모드</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normal_Mode_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>일반모드</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="normal_Mode_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QPushButton" name="mode_Select_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>모드선택</string>
    </property>
   </widget>
   <widget class="QLabel" name="normalModeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-family:'Malgun Gothic';&quot;&gt;일반모드&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:12pt; color:#00ff7f;&quot;&gt;마스크 착용 :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:12pt; color:#ff5454;&quot;&gt;마스크 미착용 :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="maskWearersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:12pt; color:#00ff7f;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="maskNotWearersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:12pt; color:#ff5454;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="normalModeLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-family:'Malgun Gothic';&quot;&gt;마스크 착용률 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskRateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:17pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="normalModeLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-family:'Malgun Gothic';&quot;&gt;1분 평균 마스크 착용률 :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskWearAverageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:17pt; color:black;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>650</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="pLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>701</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string>pLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="guard_Mode_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>320</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QPushButton" name="mode_Select_Button_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>경계모드종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="guardModeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-family:'Malgun Gothic';&quot;&gt;경계모드&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="movementLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Malgun Gothic'; font-size:16pt; color:#00ff7f;&quot;&gt;움직임 없음&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="passwordButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>입력</string>
    </property>
   </widget>
  </widget>
  <zorder>pLabel</zorder>
  <zorder>mode_Select_Widget</zorder>
  <zorder>normal_Mode_Widget</zorder>
  <zorder>endButton</zorder>
  <zorder>guard_Mode_Widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
